newrelic-security-agent/src/main/java/com/newrelic/agent/security/instrumentator/utils/AgentUtils.java

```diff
@@ -644,4 +646,10 @@ private void applyNRPolicyOverride() {
         this.setPolicyOverridden(override);
     }
 
+
+    public static void sendApplicationURLMappings() {
+        ApplicationURLMappings applicationURLMappings = new ApplicationURLMappings(URLMappingsHelper.getApplicationURLMappings());
+        logger.logInit(LogLevel.INFO, String.format("Collected application url mappings %s", applicationURLMappings.toString()), Agent.class.getName());
```

Comment:
.toString() is not required.

Reply:
Removed this in commit [710fb2b](https://github.com/newrelic/csec-java-agent/pull/51/commits/710fb2bc4bcb8d35cf9db31503e88d3aa3ce1761)